src/main/java/net/dv8tion/jda/core/events/message/MessageReceivedEvent.java

```diff
@@ -21,8 +21,10 @@
 
 /**
  * <b><u>MessageReceivedEvent</u></b><br>
- * Fired if a Message is sent in a {@link net.dv8tion.jda.core.entities.MessageChannel MessageChannel}.<br>
- * <br>
+ * Fired if a Message is sent in a {@link net.dv8tion.jda.core.entities.MessageChannel MessageChannel}.
```

Comment:
Due to the changes coming in the Message rewrite, this should probably be "if a Message is received..."